paas/install_cli.go

```diff
@@ -47,5 +48,11 @@ func downloadFuseMLCLI(ui *ui.UI) error {
 	ui.Note().Msg(fmt.Sprintf(
 		"FuseML command line client saved as %s.\nCopy it to the location within your PATH (e.g. /usr/local/bin).",
 		path))
+
+	fuseml_url := fmt.Sprintf("http://%s.%s", deployments.CoreDeploymentID, domain)
+	ui.Note().Msg(fmt.Sprintf(
+		"\nTo use the FuseML CLI, you must point it to the FuseML server URL, e.g.:\n\n    export FUSEML_SERVER_URL=%s",
```

Comment:
Just to keep it in line with the previous output, I would like to suggest removing the initial `\n`.

Reply:
done